CMakeLists.txt

```diff
@@ -559,9 +559,15 @@ set_target_properties(niflib_static
    PROPERTIES
    COMPILE_DEFINITIONS NIFLIB_STATIC_LINK)
 
-# build the tests, this needs boost
-enable_testing()
-add_subdirectory(test)
+#find boost and enable testing
+find_package(Boost 1.75.0 EXACT)
```

Comment:
I realise that this line should be `find_package(Boost)` in order to use the latest versions of boost, but I unfortunatley kept getting an error that CMake could not find boost, and this was the fix for it. This is only temporary and I'll raise an issue to fix this in the next pull request.